src/helpers/calculationStepFunctions/numericCalculationFunctions.ts

```diff
@@ -0,0 +1,40 @@
+import { create, all, string } from 'mathjs'
+import { CurrentSurveyAnswer } from '@careevolution/mydatahelps-js'
+
+export const calculateValue = (calcString :string, surveyAnswers: CurrentSurveyAnswer[]) => {
+  const config = { }
+  const math = create(all, config)
+
+  try {
+    let answerDict: {[id: string] : number} = createScope(surveyAnswers);
+    try {
+      return(math.evaluate(calcString, answerDict)._data[0].toString());
+    }
+    catch {
+      return("Calculation Error - Unmatched Variable") 
+    }
+  }
+  catch(error: unknown) {
+    if (error instanceof Error) {
+      return(error.message);
+    }
+  }
+}
+
+const createScope = (surveyAnswers: CurrentSurveyAnswer[]) => {
+  let answerDict: {[id: string] : number} = {};
+  surveyAnswers.map((answer) => {
+    if (answer.answers.length > 1) {
+      throw new Error(`Calculation Error - Calculation not available for multiple-select text choice questions (${answer.resultIdentifier})`)
```

Comment:
I think we would not want to throw errors for any of these?  We should just skip it and include the variables we can...otherwise any survey with a multi-select text choice or a "." in the resultIdentifier will not work